### PR TITLE
[Issue43] Added readme for Boolean-expressions

### PR DESCRIPTION
**Pull request by yogesh-desai**

### CODE REVIEW

File: boolean-expressions.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+---
+
+# Boolean-Expressions
+Boolean-expressions program demonstrates different boolean expressions in Go like ||,&&,!
+
+```
+/*
```
Comment (sagar-jadhav) [inline boolean-expressions.md]: Remove description from here and add above just below `# Boolean-Expressions`

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ To perform hands-on on **go-examples** it is prerequisite to have Go installed o
 ## Contents
 
 * [Arrays](./arrays.html)
-* [Boolean Expressions](https://github.com/sagar-jadhav/go-examples/blob/master/src/boolean-expressions.go)
+* [Boolean Expressions](https://github.com/sagar-jadhav/go-examples/blob/master/src/boolean-expressions.md)
```
Comment (sagar-jadhav) [inline README.md]: Update link as `./boolean-expressions.html`

**Review by sagar-jadhav — CHANGES_REQUESTED** [review]
Linking  with #43

**Comment by yogesh-desai** [comment]
@sagar-jadhav updated PR

**Review by sagar-jadhav — APPROVED** [review]

**Pull request merged**